edx_proctoring/templates/proctored_exam/id_verification.html

```diff
@@ -0,0 +1,61 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions message-left-bar" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+
+    <div class="">
+        <h3>
+            {% blocktrans %}
+                Complete verification prior to starting your proctored exam.
+            {% endblocktrans %}
+        </h3>
+        <p>
+            {% blocktrans %}
+                In order to start your proctored exam, you must successfully complete identity verification.
+            {% endblocktrans %}
+        </p>
+        {% if verification_status == 'pending' %}
+            <p>
+                {% blocktrans %}
+                    Your verification is currently pending. You should expect to hear back in 2-3 days
```

Comment:
"Your verification is pending. Results should be available 2-3 days after you submit your verification."